lib/controllers/ServiceBrokerApiController.js

```diff
@@ -71,10 +71,16 @@ class ServiceBrokerApiController extends BaseController {
       res.status(409).send({});
     }
 
+    function badRequest(err) {
+      /* jshint unused:false */
+      res.status(400).send(`{"err":{${err.message}}}`);
+    }
+
     return Promise
       .try(() => req.instance.create(params))
       .then(done)
-      .catch(ServiceInstanceAlreadyExists, conflict);
+      .catch(ServiceInstanceAlreadyExists, conflict)
+      .catch(BadRequest, badRequest);
```

Comment:
THis is not required. BadRequest is automatically sent with 400 as part of global error handler in middleware.